### DISCUSSION TITLE
Nach jedem Update muss das Limit neu gesetzt werden

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#2112] [General]
Hi,

ich habe jetzt mal die letzten vier Updates gewartet um mal nachzufragen ob es normal ist nach jedem OTA Update das Limit einstellen zu müssen.
Geschrieben wird die opendtu-generic_esp32.bin auf einen HMS-2000-4T. Das Limit wird nach dem flashen immer auf 0 gesetzt.
Dann stelle ich es wieder auf 100 permanent und es bleibt auf 100% bis zum nächsten Flashen.
Dieses Verhalten habe ich aber erst mit diesem Wechselrichter mit dem alten HM-1500 gab es das Problem einmal dann nicht mehr.
Den esp32 hatte ich auch schon komplett platt gemacht und am PC neu geflasht aber es hat nicht geholfen.
Woran könnte das liegen?

**Comment by tbnobody** [comment]
Diesen Eintrag aus den FAQ hast du aber berücksichtigt oder?
![image](https://github.com/tbnobody/OpenDTU/assets/1753222/30bdb42e-a14f-4809-819c-dc4e37796d0e)

Das Limit wird nicht sofort aus den Invertern gelesen.

**Comment by jblond4711** [comment]
Hi,

den Eintrag kannte ich nicht sollte aber nicht das Problem sein da das Limit auch nach 10 Minuten nach dem Update immer noch bei 0 ist, hätte ich im ersten Post noch schreiben sollen. Gewartet habe ich insgesamt eine Stunde und habe dann das Limit wieder auf 100% gesetzt.
Das Update selber dauert auch nur ein paar Sekunden gefühlt nicht länger als 20 Sekunden.
Wie gesagt das Limit muss ich ja nur nach einem Update neu auf 100% setzen.

**Comment by tbnobody** [comment]
Verwendest du MQTT? Ist das Limit von Null ggf. Retained im Broker? (OpenDTU setzt keinen Wert ohne externen Einfluss!)

**Comment by jblond4711** [comment]
Hi,

nein ich nutze kein MQTT. Es läuft nur OpenDTU. Der einzige Unterschied zum HM 1500 ist das jetzt der cmt2300A und nicht der nrf24 verwendet wird.
Ich war ja auch ziemlich verwundert das das Limit nach jedem Update auf 0 Gesetzt ist und bleibt.

**Reply by mroenne2022** [reply]
Ich hab das auch schon mal gedacht, aber es ist tasächlich so das man Geduld haben muss. Ich hab gestern meine "Haupt-DTU" upgedatet und heute morgen weitere 3, bei allen DTU`n hat sich die Limit Anzeige nach einigen Minuten wieder auf 100% bei allen 3 Wechselrichtern wieder korrekt dargestellt. Zunächst bleibt sie bei 0%, aber nach und nach wurde es bei den WR wieder richtig angezeigt.
